code/modules/power/engines/tesla/energy_ball.dm

```diff
@@ -231,7 +231,7 @@
 	var/mob/living/carbon/C = A
 	C.dust()
 
-/proc/tesla_zap(atom/source, zap_range = 3, power, zap_flags = ZAP_DEFAULT_FLAGS, list/shocked_targets = list())
+/obj/singularity/energy_ball/proc/tesla_zap(atom/source, zap_range = 3, power, zap_flags = ZAP_DEFAULT_FLAGS, list/shocked_targets = list(), is_energy_ball = FALSE)
```

Comment:
i don't see where is_energy_ball is being used

Reply:
Forgot to remove this!